packages/base-driver/lib/basedriver/core.js

```diff
@@ -45,16 +47,6 @@ class DriverCore {
    */
   initialOpts;
 
-  /**
-   * @type {Capabilities}
-   */
-  caps;
```

Comment:
is this a breaking change?

Reply:
AFAIK neither `DriverCore` nor `AppiumDriver` (the only other thing besides `BaseDriver` which extends it) is part of our public API, so no...